package.json

```diff
@@ -1,23 +1,46 @@
 {
   "name": "@mongodb-js/zstd",
-  "version": "1.2.2",
+  "version": "2.0.0-beta.0",
   "main": "index.js",
   "types": "index.d.ts",
   "repository": "https://github.com/mongodb-js/zstd",
   "files": [
     "index.d.ts",
     "index.js"
   ],
+  "dependencies": {
+    "bindings": "^1.5.0",
+    "node-addon-api": "^8.1.0"
+  },
   "license": "Apache-2.0",
   "devDependencies": {
+    "@mongodb-js/zstd": "^1.2.0",
+    "@typescript-eslint/eslint-plugin": "^8.11.0",
     "chai": "^4.5.0",
     "mocha": "^10.8.2",
     "prettier": "^3.3.3",
-    "standard-version": "^9.5.0"
+    "clang-format": "^1.8.0",
+    "sinon-chai": "^3.7.0",
+    "chai-subset": "^1.6.0",
+    "eslint": "^9.14.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-plugin-prettier": "^5.2.1",
+    "source-map-support": "^0.5.21"
```

Comment:
Are we going to add typescript/ts-node? That's all this dep is used for, to fix stack traces in ts-node loaded tests.

Reply:
No TS so we can remove this